ttnn/cpp/pybind11/events.cpp

```diff
@@ -5,6 +5,8 @@
 #include "events.hpp"
 
 #include "tt_metal/impl/event/event.hpp"
+#include "pybind11/operators.h"
```

Comment:
not sure this is the right include here. do you know whats missing?

Reply:
Thanks for looking it over.  Indeed that's the wrong one.  I think I got trigger happy :)

This file was complaining about unable to resolve `py::class_` and `module.def`.  Looks like `pybind11/pybind11.h` is probably the right include.  Updated.